commercial/app/model/commercial/CapiSingle.scala

```diff
@@ -0,0 +1,24 @@
+package model.commercial
+
+import common.Edition
+import common.commercial.{BrandHunter, Branding}
+import model.{ContentType, ElementsFormat, ImageElement}
+import play.api.libs.json.{Json, Writes}
+
+case class CapiSingle(articleHeadline: String, articleUrl: String,
+                      articleText: Option[String], articleImage: Seq[ImageElement],
+                      audioTag: Boolean, galleryTag: Boolean,
+                      videoTag: Boolean, branding: Option[Branding])
+
+object CapiSingle {
+  import ElementsFormat._
+
+  def fromContent(contentType: ContentType): CapiSingle = {
```

Comment:
You can, but then when it comes to using the Play JSON reads/writes macros, it can stumble when it tries to find the correct `apply`/`unapply` method to use.